clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -2206,6 +2206,21 @@ void Clang::AddRISCVTargetArgs(const ArgList &Args,
           << A->getSpelling() << Val;
     }
   }
+
+  if (Arg *A = Args.getLastArg(options::OPT_mriscv_abi_vlen_EQ)) {
+    StringRef ABIVLenStr = A->getValue();
+    unsigned ABIVLen;
+    const Driver &D = getToolChain().getDriver();
+    if (ABIVLenStr.getAsInteger(10, ABIVLen) || ABIVLen < 128 ||
```

Comment:
Why is 128? Doesn't the spec allow 32 and 64?

Reply:
Correct, it supports 32 and 64 also, I'll correct it.